frontend/src/pages/MainPage.tsx

```
@@ -0,0 +1,21 @@
+import React from 'react';
+import AuctionCard from '../components/auction/AuctionCard';
+import AuctionList from '../components/auction/AuctionList';
+
+interface MainPageProps {
+    
+}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)